diff -ruN genimage-13.1/util.c genimage-13/util.c
--- genimage-13.orig/util.c	2020-10-07 22:54:05.448271228 +0200
+++ genimage-13/util.c	2020-05-08 09:32:35.201970284 +0200
@@ -387,7 +387,7 @@
 	free(fiemap);
 
 	/* If failure is due to no filesystem support, return a single extent */
-	if (ret == -EOPNOTSUPP)
+	if ((ret == -EOPNOTSUPP) || (ret == -ENOTTY))
 		return whole_file_exent(size, extents, extent_count);
 
 	image_error(image, "fiemap %s: %d %s\n", filename, errno, strerror(errno));
